include/kframework/schedule.md

```diff
@@ -0,0 +1,348 @@
+    rule SCHEDFLAG      << MERGE >> => SCHEDFLAG << LONDON >>
+      requires notBool SCHEDFLAG ==K Ghasprevrandao
+```
+
+Fee Schedule
+------------
+
+The `Schedule` determines the constants/modes of operation for each hard fork.
+There are `ScheduleFlag`s and `ScheduleConstant`s.
+
+-   A `ScheduleFlag` is a boolean value determining whether a certain feature is turned on.
+-   A `ScheduleConst` is an `Int` parameter which is used during EVM execution.
+
+### Schedule Flags
+
+A `ScheduleFlag` is a boolean determined by the fee schedule; applying a `ScheduleFlag` to a `Schedule` yields whether the flag is set or not.
+
+```k
+    syntax Bool ::= ScheduleFlag "<<" Schedule ">>" [function, total]
+ // -----------------------------------------------------------------
+
+    syntax ScheduleFlag ::= "Gselfdestructnewaccount" | "Gstaticcalldepth" | "Gemptyisnonexistent" | "Gzerovaluenewaccountgas"
+                          | "Ghasrevert"              | "Ghasreturndata"   | "Ghasstaticcall"      | "Ghasshift"
+                          | "Ghasdirtysstore"         | "Ghascreate2"      | "Ghasextcodehash"     | "Ghasselfbalance"
+                          | "Ghassstorestipend"       | "Ghaschainid"      | "Ghasaccesslist"      | "Ghasbasefee"
+                          | "Ghasrejectedfirstbyte"   | "Ghasprevrandao"
+ // --------------------------------------------------------------------
+```
+
+### Schedule Constants
+
+A `ScheduleConst` is a constant determined by the fee schedule.
+
+```k
+    syntax Int ::= ScheduleConst "<" Schedule ">" [function, total]
+ // ---------------------------------------------------------------
+
+    syntax ScheduleConst ::= "Gzero"            | "Gbase"              | "Gverylow"      | "Glow"          | "Gmid"        | "Ghigh"
+                           | "Gextcodesize"     | "Gextcodecopy"       | "Gbalance"      | "Gsload"        | "Gjumpdest"   | "Gsstoreset"
+                           | "Gsstorereset"     | "Rsstoreclear"       | "Rselfdestruct" | "Gselfdestruct" | "Gcreate"     | "Gcodedeposit"  | "Gcall"
+                           | "Gcallvalue"       | "Gcallstipend"       | "Gnewaccount"   | "Gexp"          | "Gexpbyte"    | "Gmemory"       | "Gtxcreate"
+                           | "Gtxdatazero"      | "Gtxdatanonzero"     | "Gtransaction"  | "Glog"          | "Glogdata"    | "Glogtopic"     | "Gsha3"
+                           | "Gsha3word"        | "Gcopy"              | "Gblockhash"    | "Gquadcoeff"    | "maxCodeSize" | "Rb"            | "Gquaddivisor"
+                           | "Gecadd"           | "Gecmul"             | "Gecpairconst"  | "Gecpaircoeff"  | "Gfround"     | "Gcoldsload"    | "Gcoldaccountaccess"
+                           | "Gwarmstorageread" | "Gaccesslistaddress" | "Gaccessliststoragekey"           | "Rmaxquotient"
+ // -----------------------------------------------------------------------------------------------------------------------
```

Comment:
Move these sections to the top of the file, I think it makes more sense up there. Then the definitions of each specific schedule come after.